### PR TITLE
Rearranging emoji syntax  arguments to fix xcode 12 complaints

### PR DESCRIPTION
**Pull request by MrRipcord**

### CODE REVIEW

File: Nimble_Snapshots/PrettySyntax.swift [hunk 1]
```diff
@@ -44,11 +44,11 @@ public func == (lhs: Expectation<Snapshotable>, rhs: Snapshot) {
 // MARK: - Nicer syntax using emoji
 
 // swiftlint:disable:next identifier_name
-public func 📷(_ snapshottable: Snapshotable, file: FileString = #file, line: UInt = #line) {
-    expect(snapshottable, file: file, line: line).to(recordSnapshot())
+public func 📷(file: FileString = #file, line: UInt = #line, _ snapshottable: Snapshotable) {
```
Comment (dogo) [inline Nimble_Snapshots/PrettySyntax.swift:47]: The first parameter should be omitted to follow the old pattern. Like:
```swift
public func 📷(_ file: FileString = #file, line: UInt = #line, snapshottable: Snapshotable) {
    expect(file: file, line: line, snapshottable).to(recordSnapshot())
}
```

File: Nimble_Snapshots/PrettySyntax.swift [hunk 2]
```diff
@@ -44,11 +44,11 @@ public func == (lhs: Expectation<Snapshotable>, rhs: Snapshot) {
...
+  expect(file: file, line: line, snapshottable).to(recordSnapshot())
 }
 
 // swiftlint:disable:next identifier_name
-public func 📷(_ snapshottable: Snapshotable, named name: String, file: FileString = #file, line: UInt = #line) {
-    expect(snapshottable, file: file, line: line).to(recordSnapshot(named: name))
+public func 📷(file: FileString = #file, line: UInt = #line, _ snapshottable: Snapshotable, named name: String) {
```
Comment (dogo) [inline Nimble_Snapshots/PrettySyntax.swift:52]: Same here. Should be like:

```swift
public func 📷(_ name: String, file: FileString = #file, line: UInt = #line, snapshottable: Snapshotable) {
    expect(file: file, line: line, snapshottable).to(recordSnapshot(named: name))
}
```

**Review by dogo — COMMENTED** [review]
This PR is complementary to #206 . Thanks @MrRipcord

File: Nimble_Snapshots/PrettySyntax.swift (hunk 2 above)
Comment (MrRipcord) [reply Nimble_Snapshots/PrettySyntax.swift:52]: Thank you very much for the help @dogo !

**Pull request merged**